corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -461,6 +461,15 @@ def _get_form_ids_for_user(domain, user_id, is_deleted):
             results = fetchall_as_namedtuple(cursor)
             return [result.form_id for result in results]
 
+    @staticmethod
+    @transaction.atomic
+    def write_blob_bucket(attachment, bucket):
+        with get_cursor(XFormInstanceSQL) as cursor:
+            cursor.execute(
+                'SELECT write_blob_bucket(%s, %s::text)',
```

Comment:
From what I understand psycopg2/django infers the types from the fields in the django model. Because strings are not in the django model it's necessary to explicitly cast them. They have to be explicitly cast because the way we use sql functions there's not a direct mapping to a table that postgres can realize at query time